File: guardrails&sandbox/pi-lens/tests/fixtures/real-bugs/ruby/silent-and-security.rb
def parse_and_eval(input)
  begin
    Integer(input)
  rescue
    # BUG:silent-error empty rescue
  end

  secret = "hardcoded-ruby-secret" # BUG:secrets
  secret

  # BUG:injection eval on user input
  eval(input)
end

File: guardrails&sandbox/pi-lens/tests/fixtures/real-bugs/python/silent-and-security.py
def handle_payload(payload: str) -> None:
    try:
        int(payload)
    except Exception:
        # BUG:silent-error bare except + pass
        pass

    token = "hardcoded-token"  # BUG:secrets
    _ = token

    # BUG:injection eval from external input
    eval(payload)
